pkgs/development/python-modules/pymodes/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchPypi, buildPythonPackage, numpy, pyzmq }:
+
+buildPythonPackage rec {
+  pname = "pyModeS";
+  version = "2.11";
+
```

The source only says Python 3 but I suggest to limit it to supported Python releases. 

```suggestion
  format = "setuptools";

  disabled = pythonOlder "3.7";
```

---

Done.